### PR TITLE
keyvault-kek: implement key wrap/unwrap

### PR DESCRIPTION
**Pull request by LukeCarrier**
AVADO folks, this is related to SRE-1058

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 A great cloud needs great tools, but apparently Microsoft forgot to create either. Implements supports for the following missing operations:
```
Comment (marcusmills) [inline README.md:3]: lol

File: src/keyvault-kek/setup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+VERSION = '0.1.0'
+DEPENDENCIES = []
```
Comment (LukeCarrier) [inline src/keyvault-kek/setup.py:20]: We learned today that we require azure-cli >= 2.1.0, else you get a delightful stack trace:

```console
cannot import name 'KeyVaultAuthentication' from 'azure.keyvault' (unknown location)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/lib/python3/dist-packages/azure/cli/core/commands/__init__.py", line 608, in execute
    raise ex
  File "/usr/lib/python3/dist-packages/azure/cli/core/commands/__init__.py", line 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/lib/python3/dist-packages/azure/cli/core/commands/__init__.py", line 659, in _run_job
    six.reraise(*sys.exc_info())
  File "/usr/lib/python3/dist-packages/six.py", line 703, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/azure/cli/core/commands/__init__.py", line 636, in _run_job
    result = cmd_copy(params)
  File "/usr/lib/python3/dist-packages/azure/cli/core/commands/__init__.py", line 306, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/azure/cli/command_modules/keyvault/_command_type.py", line 96, in keyvault_command_handler
    client = client_factory(self.command_loader.cli_ctx, command_args)
  File "/usr/lib/python3/dist-packages/azure/cli/command_modules/keyvault/_client_factory.py", line 18, in keyvault_data_plane_factory
    from azure.keyvault import KeyVaultAuthentication, KeyVaultClient
ImportError: cannot import name 'KeyVaultAuthentication' from 'azure.keyvault' (unknown location)
```

File: src/keyvault-kek/setup.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+setup(
+    name='azure-cli-ext-keyvault-kek',
+    version=VERSION,
+    description='Azure Key Vault Key Encryption Key extension',
+    long_description=README,
+    license='MIT',
+    author='Luke Carrier',
+    author_email='luke.carrier@avadolearning.com',
+    url='https://github.com/AVADOLearning/az-cli-key-vault-kek',
```
Comment (LukeCarrier) [inline src/keyvault-kek/setup.py]: Wrong!

File: src/keyvault-kek/setup.py (hunk 2 above)
Comment (LukeCarrier) [reply src/keyvault-kek/setup.py:20]: Should be fixed by `azext.minCliCoreVersion`...

**Review by marcusmills — APPROVED** [review]

**Pull request merged**